modules/navier_stokes/doc/content/source/linearfvbcs/LinearFVP1RadiationMarshakBC.md

```diff
@@ -0,0 +1,27 @@
+# LinearFVP1RadiationMarshakBC
+
+This boundary condition implements the Marshak boundary conditions for
+the radiative heat flux.
+
+## Overview
+
+The Marshak boundary condition implemented reads as follows [!citep](modest2021):
+\begin{equation}
+(-\Gamma \nabla G) \cdot \vec{n_b}   = (G - 4 \sigma T_{b, rad}^4) \frac{\epsilon_b}{2 \Gamma (2 - \epsilon_b)}
```

Comment:
did you figure out where the differences with non-linear FV P1 came from?

Reply:
yes, there might be a 4 missing in the HeatConduction::Constants::sigma * Utility::pow<4>(_temperature_radiation) term:

  return _opacity(makeElemArg(_current_elem), determineState()) *
         (_var(makeElemArg(_current_elem), determineState()) -
          HeatConduction::Constants::sigma * Utility::pow<4>(_temperature_radiation(
                                                 makeElemArg(_current_elem), determineState())));